test/e2e/storage/in_tree_volumes.go

```diff
@@ -38,7 +38,6 @@ var testDrivers = []func() storageframework.TestDriver{
 	drivers.InitEmptydirDriver,
 	drivers.InitCinderDriver,
 	drivers.InitVSphereDriver,
-	drivers.InitAzureDiskDriver,
```

Comment:
@jsafrane Should this one stay?  I see that entries for Cinder and AWS are still here.

Reply:
reverted